core/src/test/java/org/testcontainers/junit/CopyFileToContainerTest.java

```diff
@@ -0,0 +1,39 @@
+package org.testcontainers.junit;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.startupcheck.MinimumDurationRunningStartupCheckStrategy;
+import org.testcontainers.utility.MountableFile;
+import java.io.File;
+import java.time.Duration;
+
+public class CopyFileToContainerTest {
+    private static String folderPath = new File("./src/test/resources/mappable-resource/").getAbsolutePath();
+    private static String containerPath = "/opt";
+    private static String fileName = "test-resource.txt";
+    public static GenericContainer container;
+
+    @Before
+    public void before() {
+        container = new GenericContainer("couchbase:latest")
```

Comment:
I think we can get away with using a smaller image here (and save some startup time). Would an `alpine` image be OK?

Reply:
we need container to be up and running to perform this test. Alpine container is exiting immediately after start. Because of this, i am unable to use alpine container

Reply:
Use the `alpine` image and provide a long running command (like `sleep`), using `withCommand()` on the container.